### PR TITLE
feat(onboarding): add messaging onboarding button

### PR DESCRIPTION
**Pull request by ameliahsu**
added messaging integration onboarding button

Button on Alerts page
Screenshot 2024-07-17 at 3 46 56 PM

Modal (in progress, will be updated to include buttons to add integrations)
Screenshot 2024-07-17 at 3 46 45 PM

### CODE REVIEW

File: static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx [hunk 1]
```diff
@@ -54,6 +59,32 @@ export default class SetupAlertIntegrationButton extends DeprecatedAsyncComponen
       return null;
     }
 
+    if (organization.features.includes('messaging-integration-onboarding')) {
```
Comment (cathteng) [inline static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx:62]: should we only show this if somebody doesn't have a messaging integration installed?

Comment (iamrajjoshi) [reply static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx:62]: We should also check for plan right?

Comment (ameliahsu) [reply static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx:62]: @cathteng yup, `SetupAlertIntegrationButton` returns null if messaging integration is already installed:
https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx#L53-L55

**Review by cathteng — COMMENTED** [review]
can we add a quick test for this that the correct version of the button is rendered when the feature flag is active? in `static/app/views/alerts/rules/issue/setupAlertIntegrationButton.spec.tsx`

**Review by cathteng — APPROVED** [review]

**Pull request merged**